instagram/src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './SearchBar.css';
+
+const SearchBar = () =>{
+    return(
+        <div className='searchBar'>
+            <div className='logo'>
+                <i className="fab fa-instagram"></i>
+                <img src='http://www.stickpng.com/assets/images/5a4e432a2da5ad73df7efe7a.png' alt='Instagram logo text'/>
+            </div>
+            <div className='search'>
+                <input/>
```

Comment:
When rendering this component the search input does not appear unless clicked on. Try adding an input of type text and a placeholder with the icon 😊